### PR TITLE
Update managing the system page

### PR DESCRIPTION
**Pull request by DaruZero**
# Description

This PR aims to update the second chapter of the Getting Started guide in order to adjust it to the new web version of the Management Console.

## Related issues

Close https://github.com/united-manufacturing-hub/MgmtIssues/issues/1041

## Checklist

*You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [contributing guidelines](https://umh.docs.umh.app/docs/development/contribute/new-content/add-documentation/) and the [code of conduct](CODE_OF_CONDUCT.md).
- [x] I have followed the [documentation](https://umh.docs.umh.app/docs/development/contribute/documentation/style/) style.

## Additional information

There is also a small change on the contributor's documentation, regarding the new way to set up the dev environment

### CODE REVIEW

File: umh.docs.umh.app/content/en/docs/getstarted/managingthesystem.md [hunk 1]
```diff
@@ -1,142 +1,75 @@
...
+On the left side of the Management Console, you can view the list of your
+instances. If you have just installed the UMH, you should see only one instance
+in the list.
+
+The status of your instance is indicated by color: green means everything is
+working properly, while yellow indicates that there may be an issue.
+
+The **Messages Received** statistic shows the number of messages received by you
+from the instance since you opened the Management Console. It is usually a good
+indicator of the health of your device and the latency of the connection between
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/getstarted/managingthesystem.md:26]: health of the companion *

**Review by JeremyTheocharis — CHANGES_REQUESTED** [review]

File: umh.docs.umh.app/content/en/docs/development/contribute/documentation/setup-environment.md [hunk 2]
```diff
@@ -26,13 +26,12 @@ using [Chocolatey](https://chocolatey.org/)).
 
 - [Git](https://git-scm.com/)
 - [Git LFS](https://git-lfs.com/)
-- [Hugo (Extended version)](https://gohugo.io/)
-- [Node.js LTS](https://nodejs.org/en/)
+- [Hugo (Extended version)](https://gohugo.io/) v0.111.3
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/development/contribute/documentation/setup-environment.md:29]: should not be part of this PR

File: umh.docs.umh.app/content/en/docs/getstarted/managingthesystem.md [hunk 3]
```diff
@@ -1,142 +1,247 @@
...
-  data collection to contextualization and visualization, or data storage. 
-  For rare cases that require a special feature, you can add it to your UMH
-  stack by creating your own custom microservice.
+## A Few Words About the Communication
 
-## 1. Install the UMH
+Now that you have connected a UMH instance to the Management Console, you might
+be curious about how the Management Console communicates with the instance.
 
+The Management Companion is a microservice that runs in conjunction with the UMH.
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/getstarted/managingthesystem.md:20]: The Management Companion is an essential component of the United Manufacturing Hub (UMH), acting as an agent for each UMH instance. Its main roles are:

Monitoring: It continuously checks the health of the UMH, its operating system, and connected devices like PLCs. If it detects issues or potential failures, it can alert your company's helpdesk system.

Management: The Companion allows users to configure and manage the UMH and its related components, enabling system control from a centralized point.

Security: Each incoming message or request, such as the addition of a new data source, is rigorously checked to ensure that only users with the right permissions can make changes. The system is designed to prevent any direct interaction with your servers to maintain security.

For detailed security information, please visit [insert link to the security page].

File: umh.docs.umh.app/content/en/docs/getstarted/managingthesystem.md [hunk 4]
```diff
@@ -1,142 +1,247 @@
...
-   
+The diagram below illustrates the communication flow between the Management
+Console and the instance:
 
-2. Follow the tutorial in the Management Console to create your first local 
-   instance of the United Manufacturing Hub. Once the installation is finished,
-   press the blue **Finish** button the right bottom corner of the Console. 
+![Communication between the Management Console and the instance](/images/getstarted/managingTheSystem/simpleInfrastructureDiagram.png?width=80%)
 
+## Overview of Your Instances
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/getstarted/managingthesystem.md:31]: add a picture :)

File: umh.docs.umh.app/content/en/docs/getstarted/managingthesystem.md [hunk 5]
```diff
@@ -1,142 +1,247 @@
...
-{{% notice info %}}
-A local installation functions similar to an installation on an external device,
-like an edge device or a VM. The benefit is, that you can easily set it up
-without any additional hardware and learn how to use it with simulated data.
+On the left side of the Management Console, you can view the list of your
+instances. If you have just installed the UMH, you should see only one instance
+in the list.
+
+The status of your instance is indicated by color: green means everything is
+working properly, while yellow indicates that there may be an issue.
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/getstarted/managingthesystem.md:37]: Orange indicates that the instance is experiencing connection issues. If an instance is experiencing connection issues for over 10 seocnds, it will be considered offline.

Comment (JeremyTheocharis) [reply umh.docs.umh.app/content/en/docs/getstarted/managingthesystem.md:37]: todo: double check

File: umh.docs.umh.app/content/en/docs/getstarted/managingthesystem.md [hunk 6]
```diff
@@ -1,142 +1,247 @@
...
+
+The **Messages Received** statistic shows the number of messages received by you
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/getstarted/managingthesystem.md:39]: Of the health of the connection to the companion.

File: umh.docs.umh.app/content/en/docs/getstarted/managingthesystem.md [hunk 7]
```diff
@@ -1,142 +1,247 @@
...
+from the instance since you opened the Management Console. It is usually a good
+indicator of the health of the Companion and the latency of the connection between
+you and the instance.
+
+## Monitoring the Instance's Status
+
+From the instance dashboard, in the overview tab, you can view the status of
+your instance. There are multiple interfaces that display the status of each
+component of the system.
+
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/getstarted/managingthesystem.md:49]: again pictures :)

File: umh.docs.umh.app/content/en/docs/getstarted/managingthesystem.md [hunk 8]
```diff
@@ -1,142 +1,247 @@
...
+### Modules
+
+A _Module_ refers to a group of workloads in the United Manufacturing Hub
+responsible for specific tasks. For example, the _Data Source_ module collects
+data from various sources and is composed of multiple microservices.
+
+In the **Modules** tab, you can view the status of each module. If a module is
+not healthy, it means that one or more of its microservices are not functioning
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/getstarted/managingthesystem.md:57]: it is not only monitoring the microservices, it is acutally monitoring way more:
- is the service still reachable?
- are any errors or warnings in the logs?
etc.

As erxample I woul use the Historian. THe Historian consists out of TimescaleDB, which itself consists out of a storage, a docker container linked to the storage, and one or multiple services making it accessible outside of the cluster. In HA setups, there will actrually one primary instance, and then multiple read replica.

File: umh.docs.umh.app/content/en/docs/getstarted/managingthesystem.md [hunk 9]
```diff
@@ -1,142 +1,247 @@
...
+If there is an overload on the device, you can view it here. An overloaded device
+is unable to handle the workload, and you should consider upgrading the device.
+
+### Data
+
+In the **Data** tab, you can see an overview of the data infrastructure, including
+the number of messages going through the message broker, those stored in the
+database, and messages received by each data source.
+
+### Connection
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/getstarted/managingthesystem.md:74]: This will be in the future called Asset Management. Maybe we can already explain its role here:

https://github.com/orgs/united-manufacturing-hub/projects/8/views/8?pane=issue&itemId=44473179

File: umh.docs.umh.app/content/en/docs/getstarted/managingthesystem.md [hunk 10]
```diff
@@ -1,142 +1,247 @@
...
 
-![Untitled](/images/getstarted/managingTheSystem/getStartedManagingMicServFinished.png)
+```bash
+export KUBECONFIG=/etc/rancher/k3s/k3s.yaml
+```
+
+Then, to get a list of pods, run:
+
+```bash
+kubectl get pods -n united-manufacturing-hub
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/getstarted/managingthesystem.md:121]: the --kubeconfig stuff needs to be added here

File: umh.docs.umh.app/content/en/docs/getstarted/managingthesystem.md [hunk 11]
```diff
@@ -1,142 +1,247 @@
...
+Use the retrieved credentials to log in.
 
-4. To activate the imported flow, simply click on the **Deploy** button located
-   at the top right of the screen. If everything is working as expected,
-   you should see green dots above the input and output. Once you have confirmed 
-   that the data is flowing correctly, you can proceed to display it in Grafana.
-   ![Untitled](/images/getstarted/managingTheSystem/getStartedManagingDeploy.png?width=75%)
+#### Access the RedPanda Console
 
+The RedPanda Console, used for managing the message broker, can be accessed at:
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/getstarted/managingthesystem.md:161]: A little bit more detail that it is used for managing the Kafka broker.

File: umh.docs.umh.app/content/en/docs/getstarted/managingthesystem.md [hunk 12]
```diff
@@ -1,142 +1,247 @@
...
+```text
+E1121 13:05:52.772843  218533 memcache.go:265] couldn't get current server API group list: the server has asked for the client to provide credentials
+error: You must be logged in to the server (the server has asked for the client to provide credentials)
+```
+
+This issue can be resolved by setting the `KUBECONFIG` environment variable. Run
+the following command:
+
+```bash
+export KUBECONFIG=/etc/rancher/k3s/k3s.yaml
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/getstarted/managingthesystem.md:202]: ah ok, here it is

File: umh.docs.umh.app/content/en/docs/getstarted/managingthesystem.md [hunk 13]
```diff
@@ -1,142 +1,247 @@
...
-   ![Untitled](/images/getstarted/managingTheSystem/getStartedManagingGrafanaImport.png?width=75%)
-3. If everything is working properly, you should now see a functional dashboard
-   with a temperature curve.
+```bash
+kubectl exec -it $(kubectl get pods -n united-manufacturing-hub -l app.kubernetes.io/component=timescaledb -o jsonpath="{.items[0].metadata.name}") -n united-manufacturing-hub -- psql -U postgres
+```
 
-   ![Untitled](/images/getstarted/managingTheSystem/getStartedManagingGrafanaDashboard.png?width=75%)
+Run SQL queries as needed. For an overview of the database schema, refer to the
+[Data Model](/docs/architecture/datamodel/database) documentation.
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/getstarted/managingthesystem.md:177]: in the future we can also add more sutff like viewieng errors, etc., but it is fine for the ucrrent getting started guide

**Review by JeremyTheocharis — CHANGES_REQUESTED** [review]

File: umh.docs.umh.app/content/en/docs/development/contribute/documentation/setup-environment.md (hunk 2 above)
Comment (DaruZero) [reply umh.docs.umh.app/content/en/docs/development/contribute/documentation/setup-environment.md:29]: Like I said, it's a small change to reflect the new environment setup methods. Since there can be other people that want to contribute to the documentation, I thought that pushing this asap would be a good idea

**Review by JeremyTheocharis — APPROVED** [review]
